### DISCUSSION TITLE
diploma thesis

### DISCUSSION THREAD

**Discussion by Paul-Absmanner** [diegomura/react-pdf#1139] [Q&A] [answered]
Hello.
I am going to write a diploma thesis and I want to make quotes out of https://react-pdf.org/rendering-process to describe the rendering-prozess. However when I am making quotes I have to state the author of site and the publishing year. Can anyone help me to get this information?
Thanks in advance
Paul

**Comment by Chagall** [comment, marked as answer]
Check this link here: https://academia.stackexchange.com/questions/14010/how-do-you-cite-a-github-repository

They have some examples on how to cite github repositores.

Hope it helps, stay safe!

**Comment by diegomura** [comment]
I think what @Chagall suggested solves this but let me know if there's anything I can do! 
Just in case thought, that section will change a bit in the new 2.0 version. You can see the changes [here](https://github.com/react-pdf/site/blob/v2/docs/rendering-process.md)
